org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIasFireIndicator.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zigbee.internal.converter;
+
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.openhab.binding.zigbee.ZigBeeBindingConstants;
+
+import com.zsmartsystems.zigbee.ZigBeeEndpoint;
+import com.zsmartsystems.zigbee.zcl.ZclCommandListener;
+import com.zsmartsystems.zigbee.zcl.clusters.iaszone.ZoneTypeEnum;
+
+/**
+ * Converter for the IAS fire indicator.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ZigBeeConverterIasFireIndicator extends ZigBeeConverterIas implements ZclCommandListener {
```

Comment:
I think that it is superfluous to add the `implements ZclCommandListener` here, as the superclass `ZigBeeConverterIas` already implements this interface. If you think that it is valuable for documentation purposes here, leave as is (my personal preference would be to remove it).

Reply:
This is no doubt a cut and paste error - I agree - it should be removed.